src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -1103,10 +1103,7 @@ public static class MenuNames
          public static readonly string NewFormulation = "Add &Formulation...";
          public static readonly string NewEvent = "Add &Event...";
          public static readonly string AddObservedData = "Add &Observed Data...";
-         public static readonly string AddAmountObservedData = "Add &Amount Observed Data...";
```

Comment:
None of that is required. Only import observed data. The selection of dimension will set the way the data is being imported

Reply:
Good idea